recipe/meta.yaml

```
@@ -1,55 +1,195 @@
 {% set name = "lumen" %}
-{% set version = "0.4.1" %}
+{% set version = "0.9.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-bundle
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 5dcb6b53bf6ce3ad8d9c881dc5325917ada258efb971f257786ef1c050668424
+  sha256: 575a9ef9444eda55a087c0201345d12ce1743f702538cf3170803a4c57d671e6
 
 build:
-  noarch: python
   number: 0
-  entry_points:
-    - lumen = lumen.command:main
-  script:
-    - {{ PYTHON }} -m pip install . -vv
-
-requirements:
-  host:
-    - python >=3.7
-    - pip
-    - param >=1.7.0
-    - pyct >=0.4.4
-    - setuptools >=30.3.0
-  run:
-    - python >=3.7
-    - panel >=0.11
-    - hvplot
-    - holoviews >=1.14
-    - pandas
-
-test:
-  imports:
-    - lumen
-  requires:
-    - pip
-  commands:
-    - pip check
-    - lumen --help
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      entry_points:
+        - lumen = lumen.command:main
+      skip: True  # [py<310]
+      script: python -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+    requirements:
+      host:
+        - python
+        - pip
+        - hatchling
+        - hatch-vcs
+      run:
+        - python
+        - panel >=1.5.0
+        - hvplot
+        - holoviews >=1.17.0
+        - pandas
+        - bokeh
+        - intake <2
+        - jinja2 >3.0
+        - numpy
+        - packaging
+        - param >=1.9.0
+        - sqlglot
+        - pydantic-extra-types
+        - jsonschema
+        - nbformat
```

```suggestion
        - pydantic-extra-types
        - jsonschema
        - nbformat
```
Not in lumen itself: https://github.com/holoviz/lumen/blob/v0.9.0/pyproject.toml#L29-L41